http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -111,6 +111,7 @@ public class Oidc {
     public static final String REDIRECT_URI = "redirect_uri";
     public static final String REFRESH_TOKEN = "refresh_token";
     public static final String RESPONSE_TYPE = "response_type";
+    public static final String SESSION_ID = "session_id";
```

Comment:
Maybe we can rename this to something like `SESSION_RANDOM_VALUE="session_random_value"`